x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/plan/logical/command/sys/SysTypes.java

```diff
@@ -81,7 +83,9 @@ public SysTypes(Source source, int type) {
 
     @Override
     public final void execute(SqlSession session, ActionListener<Page> listener) {
-        Stream<DataType> values = SqlDataTypes.types().stream();
+        Stream<DataType> values = SqlDataTypes.types().stream()
```

Comment:
Since this check occurred in a previous class, it should be extracted since otherwise its fairly obscure and repetitive.

Reply:
Fixed.